src/commands/dev-child.ts

```diff
@@ -62,6 +50,24 @@ export default defineCommand({
         )
       }
     }
+
+    process.once('unhandledRejection', () => {
+      sendIPCMessage({ type: 'nuxt:internal:dev:restart' })
```

Comment:
Perhaps we can show a clear warning about this behavior?

It looks a positive thing against unhandledRejections but also looks silent